examples/Feeds/RemoveFlightsFeedItemAttributeValue.cs

```diff
@@ -0,0 +1,307 @@
+            Dictionary<FlightPlaceholderField, FeedAttribute> feedAttributes =
+                new Dictionary<FlightPlaceholderField, FeedAttribute>();
+
+            // Loops through the feed attributes to populate the map.
+            foreach (FeedAttribute feedAttribute in googleAdsRow.Feed.Attributes)
+            {
+                switch (feedAttribute.Name)
+                {
+                    case "Flight Description":
+                        feedAttributes[FlightPlaceholderField.FlightDescription] = feedAttribute;
+                        break;
+
+                    case "Destination ID":
+                        feedAttributes[FlightPlaceholderField.DestinationId] = feedAttribute;
+                        break;
+
+                    case "Flight Price":
+                        feedAttributes[FlightPlaceholderField.FlightPrice] = feedAttribute;
+                        break;
+
+                    case "Flight Sale Price":
+                        feedAttributes[FlightPlaceholderField.FlightSalePrice] = feedAttribute;
+                        break;
+
+                    case "Final URLs":
+                        feedAttributes[FlightPlaceholderField.FinalUrls] = feedAttribute;
+                        break;
+                    // The full list of FlightPlaceholderFields can be found here
+                    // https://developers.google.com/google-ads/api/reference/rpc/latest/FlightPlaceholderFieldEnum.FlightPlaceholderField
+                    default:
+                        throw new Exception("Invalid attribute name.");
+                }
+            }
+
+            return feedAttributes;
+        }
+
+        private FeedItem RemoveAttributeValueFromFeedItem(GoogleAdsClient client, in long
+                customerId,
+            Dictionary<FlightPlaceholderField, FeedAttribute> placeholdersToFeedAttributesMap,
+            string feedItemResourceName, FlightPlaceholderField flightPlaceholderFieldName)
+        {
+            // Gets the ID of the FeedAttribute for the placeholder field.
+            long attributeId = placeholdersToFeedAttributesMap[flightPlaceholderFieldName].Id.Value;
+
+            // Retrieves the feed item and its associated attributes based on its resource name.
+            FeedItem feedItem = GetFeedItem(client, customerId, feedItemResourceName);
+
+            //Creates the FeedItemAttributeValue that will be updated.
+            FeedItemAttributeValue feedItemAttributeValue = new FeedItemAttributeValue
+            {
+                FeedAttributeId = attributeId
+            };
+
+            // Gets the index of the attribute value that will be removed.
+            int attributeIndex = GetAttributeIndex(feedItem, feedItemAttributeValue);
+
+            if (attributeIndex == -1)
+            {
+                throw new ArgumentException("No matching feed attribute found for " +
+                                            $"value '{feedItemAttributeValue}'.");
```

Comment:
Indent this to 4 spaces instead of indenting to the bracket. That's the consistent style we've been using.

Reply:
Done.